utils/templates/module-present.yml.in

```diff
@@ -0,0 +1,10 @@
+---
+- name: ${name^} present test
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  - name: Ensure $name NAME is present
+    ipa$name:
+     name: NAME
```

Comment:
The `standard password` should be added here: `ipaadmin_password: SomeADMINpassword`.

Also, it seems only a single space is used to ident the block, and we use two.